arch/arm/core/aarch32/cortex_m/Kconfig

```diff
@@ -213,6 +222,16 @@ config ARMV8_M_MAINLINE
 	  ARMv8-M Main Extension includes additional features
 	  not present in the ARMv7-M architecture.
 
+config ARMV8_1_M_MAINLINE
+	bool
+	select ARMV8_M_MAINLINE
+	help
+	  This option signifies the use of an ARMv8.1-M processor
+	  implementation, supporting the Main Extension.
+
+	  ARMv8.1-M Main Extension includes additional features
```

Comment:
I am fine with this.
In theory, the additional features should be reflected in "selection" statements in this symbol definition. But it's ok that we do not have them yet.